tests/unit/test_charm.py

```diff
@@ -158,12 +159,13 @@ def test_waits_for_config(harness, lk_client, caplog):
     caplog.clear()
     harness.update_config(
         {
-            "control-node-selector": 'gcp.io/my-control-node=""',
+            "control-node-selector": "gcp.io/my-control-node=",
```

Comment:
does the description in the config need to be updated to not include quotes in config.yaml
```
e.g. 
        node-role.kubernetes.io/control-plane=""
```

Reply:
yes thank you